integration_tests/test_domestic_migration.py

```diff
@@ -40,19 +127,43 @@ def test_individuals_move_into_new_households(simulants_on_adjacent_timesteps):
         new_household_movers = movers_into_new_households & (
             after["relationship_to_reference_person"] == "Reference person"
         )
-        assert new_household_movers.any()
+        joining_recently_created_households = movers_into_new_households & (
+            ~new_household_movers
+        )
+        fuzzy_checker.fuzzy_assert_proportion(
+            "Domestic migration joining recently created households",
+            observed_numerator=joining_recently_created_households.sum(),
+            observed_denominator=len(joining_recently_created_households),
+            # This case -- people who joined a just-created household -- should be exceedingly rare.
+            target_proportion=(
```

Comment:
Note: This is an unusual one where I think it makes sense for the target to be set on the engineering side. It can be very conservative.

Reply:
Fair enough. This one felt a little weird to me because it's not something I can imagine the RT thinking to validate ahead of time -- it's such a corner case. If I'm reading your comment correctly, that's okay -- but at the point where engineering decided to check this, they would have asked RT to specify the value?

If I'm understanding correctly, that makes sense to me; I can throw this in a YAML on the research side.

Reply:
Yes, that's exactly what I meant!